brainglobe_utils/pandas/misc.py

```diff
@@ -4,18 +4,34 @@
 
 def initialise_df(*column_names):
     """
-    Initialise a pandasdataframe with n column names
-    :param str column_names: N column names
-    :return: Empty pandas dataframe with specified column names
+    Initialise a pandas dataframe with n column names.
+
+    Parameters
+    ----------
+    *column_names : str
+        N column names
+
+    Returns
+    -------
+    pd.DataFrame
+        Empty pandas dataframe with specified column names
     """
     return pd.DataFrame(columns=column_names)
 
 
 def sanitise_df(df):
     """
-    Replaces infinite values in a dataframe with NaN
-    :param df: Any dataframe
-    :return: Dataframe with Inf replaced with NaN
+    Replaces infinite values in a dataframe with NaN.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Any dataframe
+
+    Returns
+    -------
+    pd.DataFrame
+        Dataframe with Inf replaced with NaN.
     """
     df = df.replace(np.inf, np.nan)
```

Comment:
Not sure why we have a whole function that creates a copy, assigns it to the input, then returns the input, but it was here before so I won't question it.

Though it does seem like overkill when 

```python
df.replace(np.inf, np.nan, inplace=True)
```

does literally the same thing.

Reply:
Yes it does seem like overkill! I guess it is needed elsewhere in BrainGlobe